tslint.json

```diff
@@ -11,7 +11,18 @@
         "member-access": [true, "no-public"],
         "no-empty": [true, "allow-empty-catch", "allow-empty-functions"],
         "no-string-literal": false,
-        "interface-over-type-literal": false,
+        "interface-over-type-literal": true,
+        "arrow-return-shorthand": true,
+        "no-angle-bracket-type-assertion": true,
+        "no-redundant-jsdoc": true,
+        "no-unnecessary-initializer": true,
+        "no-unsafe-finally": true,
+        "object-literal-shorthand": true,
```

Comment:
I'm surprised we don't currently use this.

Reply:
I, um, blame what I typed there on being sick. What I actually meant to say was:

I'm surprised we don't currently disable this check. I remember being frustrated with it (in either ESLint or TSLint) somewhat recently.

Reply:
I'm fine with enabling this, I guess.